### PR TITLE
Upgrade to jQuery 2.1.3

### PR DESCRIPTION
**Pull request by saponifi3d**
Cleaned up the use of jQuery in a few places, upgraded the dev version to 2.1.3 which plays a bit nicer with node / CommonJS.

### CODE REVIEW

File: test/client/base_view.test.js [hunk 1]
```diff
@@ -2,9 +2,8 @@ var App = require('../../shared/app'),
     expect = require('chai').expect,
     clientTestHelper = require('../helpers/client_test'),
     sinon = require('sinon'),
-    Model = require('../../shared/base/model')
```
Comment (mdimas) [inline test/client/base_view.test.js]: was there a missing comma here before?

Comment (saponifi3d) [inline test/client/base_view.test.js]: Yep.

File: test/client/base_view.test.js [hunk 2]
```diff
@@ -133,7 +132,7 @@ describe('Base/View', function () {
 
   describe('attachOrRender', function () {
     beforeEach(function () {
-      this.$el = $('<div/>');
+      this.$el = global.window.$('<div/>');
```
Comment (saponifi3d) [inline test/client/base_view.test.js]: @mdimas @jmerrifield - this is the source of the wrong document error. stupid document.

File: test/client/base_view.test.js (hunk 1 above)
Comment (jmerrifield) [inline test/client/base_view.test.js]: shoulda used comma-first...

**Comment by mdimas** [comment]
LGTM

**Pull request merged**